src/server/components/lists/router.ts

```diff
@@ -6,33 +6,28 @@ import {
   listsResultsController,
   listsGetPrivateBetaPage,
   listsConfirmApplicationController,
-  removeLanguageGetController
+  removeLanguageGetController,
 } from "./controllers";
 import { listsRoutes } from "./routes";
 import { csrfRequestHandler } from "server/components/cookies/helpers";
 import { ingestRouter } from "server/components/lists/controllers/ingest/router";
 import annualReviewRouter from "server/components/annual-review/router";
+import { redirectToNewUrlStructure } from "./helpers";
 
 export const listsRouter = express.Router();
 
 listsRouter.get(listsRoutes.finder, csrfRequestHandler, listsGetController);
 listsRouter.post(listsRoutes.finder, csrfRequestHandler, listsPostController);
 listsRouter.get(listsRoutes.removeLanguage, csrfRequestHandler, removeLanguageGetController);
-listsRouter.get(
-  listsRoutes.results,
-  csrfRequestHandler,
-  listsResultsController
-);
+listsRouter.get(listsRoutes.results, redirectToNewUrlStructure);
```

Comment:
you can actually implement middleware handling with express Router itself which I think would be a nicer way of handling this as the old handler in the controller is not really needed anymore.  See the docs here where you can use router.route(....) to detect the presence of the country and / or service type query params and enforce the redirection to the listsResultsController : https://expressjs.com/en/4x/api.html#router.route.  See here for an existing example implementing a redirect detecting the presence of listId and listItemId:  https://github.com/UKForeignOffice/lists/blob/7c1a7dce495f8ffbd8fa68468fb5b230e3013133/src/server/components/dashboard/listsItems/itemsRouter.ts#L27